cfonts/core.py

```diff
@@ -354,7 +355,14 @@ def render(
         if output:
             output[0] = style.open + output[0]
             output[-1] += style.close
-    return "\n".join(output)
+    # easier to be before copyable, so you don't have to loop
+    output = "\n".join(output)
```

Comment:
Avoid assigning the variable with a value of a different type from what is declared, otherwise mypy won't be happy.

After: str = Before: list